### ISSUE TITLE
Super Game Boy games do not run

### ISSUE THREAD

**Issue by freq-mod** [rg35xx-cfw/Koriki#47]
### What happened?

SGB enhanced gb roms appear correctly as a SimpleMenu category, yet trying to run them results in instants crash to menu.
They run through RetroArch app fine, I noticed that SimpleMenu attempts to load them via msgb_libretro core, which is something I never heard about before 🤔 Worse is, Im unable to change it to Gambatte or anything working

**Comment by acmeplus** [comment]
Yes, that's not properly configured and will be fixed in a new release.

If you want to fix it now, you can edit the file ``system/.simplemenu/section_groups/consoles.ini`` and change the [SGB] section to this:
```
[SGB]
execs = /userdata/system/.simplemenu/launchers/mesen-s_libretro,/userdata/system/.simplemenu/launchers/mgba_libretro
romDirs = /userdata/roms/sgb/
romExts = .gb,.gbc,.zip,.7z
```
